mongo/change_stream.go

```diff
@@ -457,20 +457,47 @@ func (cs *ChangeStream) ResumeToken() bson.Raw {
 }
 
 // Next gets the next result from this change stream. Returns true if there were no errors and the next
-// result is available for decoding.
+// result is available for decoding. Next blocks until an event is available for decoding or ctx expires.
+// If the given context expires during execution, the stream's error will be set and the change stream may be in an
+// invalid state and should be re-created. If Next returns false and an error occurred (i.e. cs.Err() != nil),
```

Comment:
In Next returns false and no error occurred (i.e. cursor is exhausted?) should it be called again?

Reply:
Probably not - changed to remove the part about an error.